src/bin/main/mainLoader.ts

```diff
@@ -152,6 +153,8 @@ ipcRenderer.on(
     if (partition) {
       mainWindow.partition = partition
     }
+
+    mainWindow.useragent = navigator.userAgent
```

Comment:
`mainWindow` is oriented to users. Setting `userAgent` here takes no effect to the game(and nobody would read `userAgent` here).